util/stmtsummary/statement_summary.go

```diff
@@ -579,6 +581,7 @@ func newStmtSummaryByDigestElement(sei *StmtExecInfo, beginTime int64, intervalS
 	ssElement := &stmtSummaryByDigestElement{
 		beginTime: beginTime,
 		sampleSQL: formatSQL(sei.OriginalSQL),
+		hints:     sei.Hints,
```

Comment:
What if the first statement has no hints and the second statement has hints? In this way, it doesn't record any hints. Is that expected?

Reply:
The hints are recovered from the plan which is a normalized plan including extra table hints. If a statement has a query plan then a normalized hint will be generated.